test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework/providers/gce"
 	e2eresource "k8s.io/kubernetes/test/e2e/framework/resource"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	e2emanifest "k8s.io/kubernetes/test/e2e/manifest"
```

Comment:
I would vote to alias this package as something like `manifestutils` or `testmanifests` or something of the sort.
Up to now we have been using the `e2e` prefix for framework subpackages.
I.e., if it the package was in `k8s.io/kubernetes/test/e2e/framework/manifest` then that one would be `e2emanifest`.